apps/ae_socket_connector/lib/socket_connector.ex

```diff
@@ -1008,9 +1009,19 @@ defmodule SocketConnector do
     {poi, %__MODULE__{state | round_and_updates: Map.put(state.round_and_updates, round, update_new)}}
   end
 
+  # can be removed once this is fixed
```

Comment:
Should we make a separate issue for it, just not to forget to remove it?

Reply:
When i look at the code i think it should probably stay, i'll rephrase my comment.